app/services/atmosphere/cep/esper.rb

```diff
@@ -0,0 +1,18 @@
+module Atmosphere
+  class Cep::Esper
+    def add_event_type(ev_type_id, ev_definition)
```

Comment:
Unused method argument - `ev_type_id`. If it's necessary, use `_` or `_ev_type_id` as an argument name to indicate that it won't be used. You can also write as `add_event_type(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `ev_definition`. If it's necessary, use `_` or `_ev_definition` as an argument name to indicate that it won't be used. You can also write as `add_event_type(*)` if you want the method to accept any arguments but don't care about them.